c7n/resources/ecs.py

```diff
@@ -247,6 +248,37 @@ class ServiceTaskDefinitionFilter(RelatedTaskDefinitionFilter):
     """
 
 
+@Service.action_registry.register('modify')
+class UpdateService(BaseAction):
+    """Update service(s)."""
+
+    permissions = ('ecs:UpdateService',)
+
+    def validate(self):
+        api_call_shape = self.data.get('modify')
+        if not api_call_shape:
+            return
+        api_call_shape['service'] = 'PlaceHolderService'
+        aws.shape_validate(api_call_shape, 'UpdateServiceRequest', 'ecs')
+
+    def process(self, resources):
+        client = local_session(self.manager.session_factory).client('ecs')
+        for r in resources:
+            api_call_param = {}
+            requested_change_param = self.data.get('modify')
```

Comment:
please don't make the user type in the same string multiple times to mean different things. i noticed in this pattern in your accelerator pull request, where the same word was being spelled out three times in a row. here your asking the user to say modify, modify... use attributes, update, values, frankly anything else to distinguish.